phantazm-neuron-minestom/src/main/java/com/github/phantazmnetwork/neuron/bindings/minestom/entity/MinestomDescriptor.java

```diff
@@ -0,0 +1,91 @@
+package com.github.phantazmnetwork.neuron.bindings.minestom.entity;
+
+import com.github.phantazmnetwork.commons.vector.Vec3D;
+import com.github.phantazmnetwork.commons.vector.Vec3I;
+import com.github.phantazmnetwork.neuron.agent.Agent;
+import com.github.phantazmnetwork.neuron.agent.PhysicalDescriptor;
+import com.github.phantazmnetwork.neuron.engine.PathContext;
+import com.github.phantazmnetwork.neuron.navigator.Controller;
+import com.github.phantazmnetwork.neuron.navigator.Navigator;
+import com.github.phantazmnetwork.neuron.node.Calculator;
+import com.github.phantazmnetwork.neuron.node.NodeTranslator;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.instance.Instance;
+import org.jetbrains.annotations.NotNull;
+
+import javax.naming.ldap.Control;
```

Comment:
never seen this package in my life before

Reply:
intelliJ tab completion and autoimport strikes again